rbac/management/audit_log/model.py

```diff
@@ -58,3 +62,51 @@ class AuditLog(TenantAwareModel):
     resource_id = models.IntegerField(null=True)
     action = models.CharField(max_length=32, choices=ACTION_CHOICES)
     tenant = models.ForeignKey(Tenant, on_delete=models.SET_NULL, null=True)
+
+    def get_resource_item(self, r_type, request, *args, **kwargs):
+        """Find related information (eg, name, id, etc...) for each resource item."""
+        if r_type == AuditLog.ROLE:
+            if request.data != {}:
+                role_object = get_object_or_404(Role, name=request.data["name"])
```

Comment:
Might need the tenant info or it might get roles with same name from different tenants